src/main/java/org/czekalski/petapiintegration/apiclient/v1/mapper/AnimalDtoToAnimalMapper.java

```diff
@@ -0,0 +1,23 @@
+package org.czekalski.petapiintegration.apiclient.v1.mapper;
+
+import org.czekalski.petapiintegration.apiclient.v1.dto.AnimalDto;
+import org.czekalski.petapiintegration.resourceserver.model.Animal;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AnimalDtoToAnimalMapper implements Mapper<AnimalDto, Animal> {
+    @Override
+    public Animal map(AnimalDto animalDto) {
+        if (animalDto == null) {
+            return null;
+        } else {
+            Animal animal = new Animal();
+            animal.setId(animalDto.getId());
+            animal.setName(animalDto.getName());
+            animal.setBreed(animalDto.getBreeds().getPrimary());
```

Comment:
Jak breed przyjdzie `null` to poleci NPE

Reply:
Poprawione: 7965fc0e86c5e0d348cccf5a67570ae453b958c2